modules/aws-exfiltration-protection/workspace.tf

```diff
@@ -1,14 +1,23 @@
 resource "databricks_mws_networks" "this" {
-  provider           = databricks.mws
```

Comment:
why do we need this? Default provider should work

Reply:
removed provider to use the default